src/openrct2/ride/thrill/3dCinema.cpp

```diff
@@ -11,6 +11,8 @@
 #include "../../interface/Viewport.h"
 #include "../../paint/Paint.h"
 #include "../../paint/support/WoodenSupports.h"
+#include "../../paint/tile_element/Segment.h"
+#include "../../paint/track/Segment.h"
```

Comment:
In flat rides and such these entries aren't actually used. Are they just here for consistency sake?

Reply:
Good catch. They do use `kSegmentsAll`, though, but I can still drop `track/Segment.h`.